### DISCUSSION TITLE
"direct" variables?

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [corn-config/corn#12] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68589851/198608454-fab795d6-5f18-4595-92e6-f62eea48e1ce.png)
I wanna do smth like this, but it seems like I can't?

**Comment by JakeStanger** [comment, marked as answer]
Hey, I think what you're trying to do is possible but you're just getting the syntax wrong for that ironbar module; you need to put each section inside a format.

Corn inputs can be any supported type, and will expand like that, but you need to put them in an array.

For example:

```corn
$sys_info = { format = [ $cpu_sys $memory_sys $network_sys ] }
```
